src/components/Popup.vue

```diff
@@ -17,7 +17,9 @@
                 @click.prevent="close"
             >
                 <slot name="close">
-                    <span class="icon-cancel"></span>
+                    <div class="popup__close-default-icon">
+                        <img src="../assets/icons/close.svg">
```

Comment:
`popup__close-default-icon` keby dame rovno classu `popup__icon-close` na img, ci nac ten div okolo, nie je to ani BEM ten vnoreny img styl + img este daj display block - inak pridava taku mini medzierku do divu